jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -335,8 +335,15 @@ class DeploymentFlow extends React.Component {
       credential: this.state.credential,
       region: this.state.region
     };
-    if (this.state.sshKey) {
-      args.config['authorized-keys'] = this.state.sshKey;
+    if (this.state.sshKeys.length) {
+      // Attempt to provide at least one key in case the addSSHKeys call fails.
+      args.config['authorized-keys'] = this.state.sshKeys[0].text;
```

Comment:
I'm not sure this is necessary but because adding multiple is a no-op anyways 🤷‍♂️